db/migrate/20180913035732_create_departments.rb

```diff
@@ -0,0 +1,19 @@
+class CreateDepartments < ActiveRecord::Migration[5.2]
+
+
+  def up
```

Comment:
Please fix this file according to previous instructions.

Reply:
as it is a migration file, i wont be able to make changes to it